presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                     +                                          +                                            +thread to increase CPU utilization with the cost of +
++                                     +                                          +                                            +increased memory usage.                             +
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++   ``task_hash_build_concurrency``   +               ``Integer``                +    ``task.default-concurrency`` (``1``)    +**Experimental.** Default number of local parallel  +
++                                     +                                          +                                            +hash build jobs per worker. Same as                 +
++                                     +                                          +                                            +``task_join_concurrency`` but it is used for        +
++                                     +                                          +                                            +building hashes. The value is always rounded down   +
++                                     +                                          +                                            +to the power of 2 so it's recommended to use such   +
++                                     +                                          +                                            +value in order to avoid unexpected behavior.        +
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++  ``task_aggregation_concurrency``   +               ``Integer``                +    ``task.default-concurrency`` (``1``)    +**Experimental.** Default number of local parallel  +
++                                     +                                          +                                            +aggregation jobs per worker. Same as                +
++                                     +                                          +                                            +``task_join_concurrency`` but it is used for        +
++                                     +                                          +                                            +aggregation.                                        +
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++        ``task_writer_count``        +               ``Integer``                +       ``task.writer-count`` (``1``)        +Describes how many parallel writers may try to      +
```

Comment:
Don't repeat yourself, just point to task.writer-count's config property description